mgl64/matstack/transformstack_test.go

```diff
@@ -1,4 +1,4 @@
-// This file is generated from mgl32/matstack/transformstack_test.go; DO NOT EDIT
+// This file is generated from mgl32/matstack\transformstack_test.go; DO NOT EDIT
```

Comment:
Hmm, I take it this was generated on Windows. Oh dear. Probably need a bugfix to `codegen.go`. Can you amend the commit to remove these?

Reply:
> Probably need a bugfix to `codegen.go`.

Made #80